server/views/partials/expectedArrivalCardMacro.njk

```diff
@@ -43,7 +43,7 @@
     <div class='app-card__content'>
         <div class="app-card app-card--with-aside">
             <aside class="app-card__aside">
-                <img src="{{ '/prisoner/' + arrival.prisonNumber + '/image' if arrival.prisonNumber else '/assets/images/placeholder-image.png' }}" class="app-card__image" alt="{{ arrival.lastName }}, {{ arrival.firstName }}" data-qa="prisoner-image"/>
+                <img src="{{ '/prisoners/' + data.potentialMatches[0].prisonNumber + '/image' if data.potentialMatches[0].prisonNumber else '/assets/images/placeholder-image.png' }}" class="app-card__image" alt="{{ arrival.lastName }}, {{ arrival.firstName }}" data-qa="prisoner-image"/>
```

Comment:
fix images for arrivals so we look at the potential match rather than basm prison number